tgui/packages/tgui/interfaces/CryopodConsole.js

```diff
@@ -33,7 +30,7 @@ const CrewList = (props, context) => {
       {!frozen_crew.length ? (
         <NoticeBox>No stored crew!</NoticeBox>
       ) : (
-        <Section height={10} fill scrollable>
+        <Section height={"100%"} scrollable>
```

Comment:
Section cannot fill if it's inside the collapsible. You need to implement this in some other way if you want to have a scrollbar.

Reply:
When I didn't have height="100%" the section had a height of 0 for whatever reason and wouldn't render properly.

I'll look into this more when I can, possibly not needing a collapsible anymore as it's only showing that one piece of info now.

Reply:
No crew:
![image](https://user-images.githubusercontent.com/24975989/118301763-59a8bc00-b4db-11eb-8c65-0e2eacae8ca3.png)

With crew:
![image](https://user-images.githubusercontent.com/24975989/118301950-8b218780-b4db-11eb-8a30-2b403c84bd4f.png)